### ISSUE TITLE
error: auto_detect_line_endings is deprecated

### ISSUE THREAD

**Issue by Tsjippy** [10up/plugin-check#1368]
I get this warning when checking a plugin:

```
[20-Jun-2026 05:43:55 UTC] You have an error notice: "auto_detect_line_endings is deprecated" in file "D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\vendor\squizlabs\php_codesniffer\src\Runner.php" at line: "310" . 
[20-Jun-2026 05:43:55 UTC] 
1) D:\LocalWebsites\sim-nigeria\app\public\wp-admin\admin-ajax.php(192): do_action('wp_ajax_plugin_...')
2) D:\LocalWebsites\sim-nigeria\app\public\wp-includes\plugin.php(522): WP_Hook->do_action(Array)
3) D:\LocalWebsites\sim-nigeria\app\public\wp-includes\class-wp-hook.php(365): WP_Hook->apply_filters('', Array)
4) D:\LocalWebsites\sim-nigeria\app\public\wp-includes\class-wp-hook.php(341): WordPress\Plugin_Check\Admin\Admin_AJAX->run_checks('')
5) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Admin\Admin_AJAX.php(306): WordPress\Plugin_Check\Checker\Abstract_Check_Runner->run()
6) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Checker\Abstract_Check_Runner.php(445): WordPress\Plugin_Check\Checker\Checks->run_checks(Object(WordPress\Plugin_Check\Checker\Check_Context), Array, Object(WordPress\Plugin_Check\Checker\AJAX_Runner))
7) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Checker\Checks.php(43): array_walk(Array, Object(Closure))
8) [internal function]: WordPress\Plugin_Check\Checker\Checks->{closure:WordPress\Plugin_Check\Checker\Checks::run_checks():45}(Object(WordPress\Plugin_Check\Checker\Checks\General\I18n_Usage_Check), 'i18n_usage')
9) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Checker\Checks.php(46): WordPress\Plugin_Check\Checker\Checks->run_check_with_result(Object(WordPress\Plugin_Check\Checker\Checks\General\I18n_Usage_Check), Object(WordPress\Plugin_Check\Checker\Check_Result))
10) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Checker\Checks.php(81): WordPress\Plugin_Check\Checker\Checks\Abstract_PHP_CodeSniffer_Check->run(Object(WordPress\Plugin_Check\Checker\Check_Result))
11) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\includes\Checker\Checks\Abstract_PHP_CodeSniffer_Check.php(116): PHP_CodeSniffer\Runner->runPHPCS()
12) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\vendor\squizlabs\php_codesniffer\src\Runner.php(76): PHP_CodeSniffer\Runner->init()
13) D:\LocalWebsites\sim-nigeria\app\public\wp-content\plugins\plugin-check\vendor\squizlabs\php_codesniffer\src\Runner.php(310): ini_set('auto_detect_lin...', true)
14) [internal function]: TSJIPPY\printError(8192, 'auto_detect_lin...', 'D:\\LocalWebsite...', 310)
```
